neurom/core/types.py

```diff
@@ -47,25 +47,29 @@ class NeuriteIter(OrderedEnum):
     NRN = 2
 
 
-class CompositeType(int):
-    """A composite type is a list of section types.
+class Subtypes:
 
-    Each subtype corresponds to a homogeneous subtree in the morphology.
+    def __init__(self, *subtypes):
+        self.subtypes = subtypes
```

Comment:
Do we care about the subtype order? If not you could use a `set()` and simplify the `__eq__` method.

Reply:
I believe so. An axon-carrying-dendrite has to have a basal before an axon in the traversal order.

Reply:
Ah ok
But then in the `__eq__` method we don't care about the order? From a biological point of view, `axon == [basal, axon]` should be considered the same as `axon == [axon, basal]`?